system-addon/content-src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -24,9 +24,23 @@ class ContextMenu extends React.Component {
   componentDidUnmount() {
     window.removeEventListener("click", this.hideContext);
   }
-  onKeyDown(index) {
-    if (index > this.props.tabbableOptionsLength) {
-      this.hideContext();
+  onKeyDown(event, option) {
+    switch (event.key) {
+      case "Tab":
+        if (event.shiftKey) {
+          if (option.first) {
+            this.hideContext();
+          }
+          return;
+        }
+        if (option.last) {
+          this.hideContext();
```

Comment:
I had this originally, but if you're on the last option and you press Shift + Tab, it closes the context menu because it'll satisfy the ```option.last``` criteria, when really what you want is for it to start going back up :(